src/robusta/core/discovery/utils.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+from kubernetes.client import V1Deployment, V1DaemonSet, V1StatefulSet, V1Job, V1Pod, V1ResourceRequirements
+
+from ...core.model.pods import PodResources, ResourceAttributes
+
+# This file contains utility related to k8s python api objects (rather than hikaru)
+
```

Comment:
I think that instead of calling it util.py, it is better to just make it a Discovery class and then move _create_service_info,  discover_services, __to_node_info, and discover_nodes here. (__publish_new_services and __publish_new_nodes should stay where they are)

Reply:
Fixing